jreleaser.yml

```diff
@@ -54,3 +54,18 @@ packagers:
     skipTemplates:
       - cask.rb.tpl
       - README.md.tpl
+
+  scoop:
+    active: RELEASE
+    repository:
+      active: RELEASE
+      owner: jenkins-infra
+      name: plugin-modernizer-bucket  
```

Comment:
Can a bucket store more apps ? If yes perhaps we need to ask a repo with something more generic ?

Perhaps 'scoop-bucket' ? What do you think ?